string config_path  # path to yaml file
---
ocs2_msgs/mpc_target_trajectories target_trajectory
